events/syscall/unexpected_k8s_nodeport_connection.go

```diff
@@ -0,0 +1,58 @@
+//go:build linux
+// +build linux
+
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+	"strconv"
+    "strings"
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	UnexpectedK8sNodePortConnection,
+    events.WithDisabled(), // this rule is not included in falco_rules.yaml (stable rules), so disable the action     
+)
+
+func UnexpectedK8sNodePortConnection(h events.Helper) error {
+    if h.InContainer() {
+        port := 31000
+
+        // Get the host's eth0 IP address
+        cmd := exec.Command("ip", "route", "show", "default")
+        hostEth0IP, err := cmd.Output()
+		if err != nil {
+			h.Log().WithError(err).Error("Failed to get host's eth0 IP")
+			return err
+		}
+
+		// Extract the IP address from the output
+		hostIP := strings.TrimSpace(strings.Split(string(hostEth0IP), " ")[2])
+
+		cmd = exec.Command("nc", hostIP, strconv.Itoa(port), "<", "/dev/null")
+		err = cmd.Run()
+		if err != nil {
+			h.Log().WithError(err).Error("Failed to establish NodePort connection with netcat")
+			return err
+		}
+
+		h.Log().Infof("Unexpected K8s NodePort Connection")
+		}
```

Comment:
please indent the code correctly. Use go fmt pls :pray: 

https://go.dev/blog/gofmt